gitter.py

```diff
@@ -307,7 +315,8 @@ def rooms(self):
         for json_room in json_rooms:
             if not json_room['oneToOne']:
                 log.debug("found room %s (%s)" % (json_room['name'], json_room['uri']))
-                rooms.append(GitterRoom(self, json_room['id'], json_room['uri'], json_room['name']))
+                rooms.append(GitterRoom(self, json_room['id'], json_room['uri'],
+                                        json_room['name'], json_room['topic']))
```

Comment:
`GitterRoom` doesn't have a 4th parameter (I believe you forgot to remove the `json_room['topic']` addition which was added in a previous iteration).